test/snapshot_test.rb

```diff
@@ -0,0 +1,143 @@
+require_relative 'spec_helper'
+require 'yast'
+require 'dbus'
+require_relative '../src/lib/snapper/snapshot.rb'
+
+describe Yast2::Snapper::Snapshot, "with default strategy" do
```

Comment:
what is point of documenting it with default strategy? To be honest I foudn this "strategy" think quite confusing. For me strategy more looks like proxy to snapper or communication API